lib/msal-browser/src/utils/BrowserUtils.ts

```diff
@@ -5,20 +5,37 @@
 import { INetworkModule } from "@azure/msal-common";
 import { FetchClient } from "../network/FetchClient";
 import { XhrClient } from "../network/XhrClient";
+import { BrowserAuthError } from "../error/BrowserAuthError";
 
 /**
  * Utility class for browser specific functions
  */
 export class BrowserUtils {
 
-    // #region Window Navigation
+    // #region Window Navigation and URL management
 
     /**
      * Used to redirect the browser to the STS authorization endpoint
      * @param {string} urlNavigate - URL of the authorization endpoint
      */
-    static navigateWindow(urlNavigate: string): void {
-        window.location.assign(urlNavigate);
+    static navigateWindow(urlNavigate: string, noHistory?: boolean): void {
```

Comment:
This can be done incrementally, but I think we should have an interface for navigating a browser window, since it will be different in Node, for example.

Reply:
I think it's better to leave that entirely in the browser/node packages rather than exposing an interface in common. There is too much that goes into managing the UI that is platform specific.

Reply:
Although I may just be overthinking the node vs browser implementation differences.

Reply:
I was thinking generally about an interface for "I want to send the user to this url" and then have each platform implement how that actually happens.

Reply:
Although the `noHistory` nuance does make that a little more complicated, I suppose.